### ISSUE TITLE
About LightGCN-single

### ISSUE THREAD

**Issue by LiuHDme** [kuandeng/LightGCN#2]
Hi, thanks for you job.
I'm trying to reproduce LightGCN-single 2 which only use E^(2) for final prediction. However, I only get 0.0396 on amazon-book where your report is about 0.045+ in section 4.4.1

**Details:**
The command I use is:
`python LightGCN.py --dataset amazon-book --regs [1e-4] --embed_size 64 --layer_size [64,64] --lr 0.001 --batch_size 4096 --epoch 1000`
and I **only changed 3 line** of your code in method `_create_lightgcn_embed`:
`# all_embeddings=tf.stack(all_embeddings,1)`
`# all_embeddings=tf.reduce_mean(all_embeddings,axis=1,keepdims=False)`
`u_g_embeddings, i_g_embeddings = tf.split(all_embeddings[-1], [self.n_users, self.n_items], 0)`

Is there anything I got wrong? Thanks.

**Comment by kuandeng** [comment]
I think your modification is right. You can try this command：
cd LightGCN-master/ && python setup.py build_ext --inplace && python LightGCN.py --dataset amazon-book --regs [1e-5] --embed_size 64 --layer_size [64,64] --lr 0.001 --batch_size 2048 --epoch 1000.
